Microsoft.Azure.Cosmos/src/Query/Core/SqlQuerySpec.cs

```diff
@@ -69,12 +82,37 @@ public SqlParameterCollection Parameters
             }
         }
 
+        /// <summary>
+        /// Gets or sets the boolean value that indicates whether the query should be executed as pass-through..
+        /// </summary>
+        /// <value>A boolean value that indicates whether the query should be executed as pass-through.</value>
+        [DataMember(Name = "passThrough")]        
+        public bool PassThrough
```

Comment:
Just use auto property. No reason to define a private variable. 
```suggestion
        public bool PassThrough {get; set;}
```